interchain-token-service/src/primitives.rs

```diff
@@ -26,7 +34,7 @@ pub enum TokenManagerType {
 /// ITS message type that can be sent between ITS contracts for transfers/token deployments
 /// `ItsMessage` that are routed via the ITS hub get wrapped inside `ItsHubMessage`
 #[cw_serde]
-#[derive(Eq)]
+#[derive(Eq, strum::IntoStaticStr)]
```

Comment:
Why do we need that?

Reply:
In the event to make it more readable
https://github.com/axelarnetwork/axelar-amplifier/pull/572/files#diff-3fafd412ea6177bfd7b62305e64a5e58a51dd84ed8db07fe15d118c30c95cf9dR49